test/Microsoft.Framework.Runtime.Tests/ServicingFacts.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Security.AccessControl;
+using Xunit;
+
+namespace Microsoft.Framework.Runtime.Tests
+{
+    public class ServicingFacts
```

Comment:
Unit tests shouldn't touch the disk, this looks like a functional test, put it in that assembly.